client/src/components/post/Comments.js

```diff
@@ -26,8 +33,13 @@ const Comments = (props) => {
                     </h6>
                   </Link>
                 </div>
-                <div className='align-self-end'>
-                  <p>{convertDate(comment.created_at)}</p>
+                <div className='d-flex flex-column '>
+                  <p className='my-0 small-font'>
+                    {convertDate(comment.created_at)}
+                  </p>
+                  {loggedInUser.id === comment.user_id ? (
+                    <ContentOptions id={comment.id} contentType='comment' />
+                  ) : null}
```

Comment:
i know your prettier probably did this but its hard to read, could you put this ternary on a single line?